catkin_ws/src/cucumber_msgs/srv/HarvestAction.srv

```diff
@@ -9,9 +9,13 @@ byte status
 
 # Possible status codes for the harvest process.
 byte OK = 0
-byte GRAB_ERR = 1
-byte CUTT_ERR = 2
-byte DROP_ERR = 3
-byte MOVE_ERR = 4
-byte VACU_ERR = 5
-byte NOVAC_ERR = 6
+byte ERROR = 1
+byte GRAB_ERR = 2
+byte VACU_ERR = 3
+byte CUTT_ERR = 4
+byte MOVE_ERR = 5
+
+# Deprecated constants, do not use these!
+byte DROP_ERR = 7
+byte NOVAC_ERR = 8
+
```

Comment:
Shouldn't we keep the error codes that already exist the same values? Otherwise error codes that have been serialized or logged anywhere don't refer to the same error anymore.